doc/api/fs.md

```diff
@@ -725,19 +725,55 @@ added: v0.8.6
 -->
 
 * `fd` {Integer}
-* `len` {Integer}
+* `len` {Integer} default = `0`
 * `callback` {Function}
 
 Asynchronous ftruncate(2). No arguments other than a possible exception are
 given to the completion callback.
 
+If the file referred by the file descriptor has more number of bytes than the
+`len` then, only the first `len` bytes will be retained in the file.
```

Comment:
```
If the file referred to by the file descriptor was larger than `len` bytes, only the first `len` bytes will be retained in the file.
```
How about that (it’s only a minor difference to what you wrote)?

The second paragraph below could probably be written quite closely to the man page, e.g.
```
If the file previously was shorter than `len` bytes, it is extended, and the extended part reads as null bytes ('\0').
```